corehq/apps/app_manager/dbaccessors.py

```diff
@@ -500,16 +501,24 @@ def get_version_build_id(domain, app_id, version):
 def get_case_types_from_apps(domain):
     """
     Get the case types of modules in applications in the domain.
+    Also returns case types for SaveToCase properties in the domain, if the toggle is enabled.
     :returns: A set of case_types
     """
```

Comment:
I can't comment on the logic itself, but I'll offer a nitpick on the code:

To make it easier to read, I'd split out this change into its own function. Something like
```python
save_to_case_case_types = _get_save_to_case_case_types(domain) # ugh, naming is hard
```